lib/portlayer/exec/container.go

```diff
@@ -596,6 +596,8 @@ func (c *Container) Remove(ctx context.Context, sess *session.Session) error {
 				log.Errorf("Error while attempting to unregister container VM: %s", err)
 				return err
 			}
+		case *types.ConcurrentAccess:
```

Comment:
is this error happens during vm delete or unregister? I think that's in unregister if you're using llpm, but not sure for here.
btw, if this error happens, do you need a return here?